### ISSUE TITLE
Finding nodes by type or id

### ISSUE THREAD

**Issue by thomas-blackbird** [meyfa/php-svg#40]
Does it exists a feature which allows to search nodes by type and id in the svg dom?

**Comment by meyfa** [comment]
There is no such feature as of yet, but it would be a good addition. I will try to draft something up as soon as possible.

**Comment by meyfa** [comment]
I implemented `getElementById($id)` (936d22c2c3c104bf08f1926eab51379158e528e6) and `getElementsByTagName($tagName)` (2283a216a32493450adf1b35462e985767b5bf54). They behave exactly like their JavaScript counterparts.

Example:

```php
$image = SVGImage::fromFile('file.svg');
$doc = $image->getDocument();

$byId = $doc->getElementById('foobar');
// $byId is now set to the element with id="foobar"

$byTagName = $doc->getElementsByTagName('rect');
// $byTagName is now set to an array with all <rect /> nodes
```

Note that `getElementById` can only be invoked on the document (as usual, since ids are unique across the document). `getElementsByTagName` can be invoked on any node.
